src/openassetio-python/cmodule/src/hostApi/ManagerBinding.cpp

```diff
@@ -136,9 +136,13 @@ void registerManager(const py::module& mod) {
             return self.resolve(entityReferences, traitSet, context);
           },
           py::arg("entityReferences"), py::arg("traitSet"), py::arg("context").none(false))
-      .def("getWithRelationship", &Manager::getWithRelationship, py::arg("entityReferences"),
-           py::arg("relationshipTraitsData").none(false), py::arg("context").none(false),
-           py::arg("successCallback"), py::arg("errorCallback"),
+      .def("getWithRelationship",
+           py::overload_cast<const EntityReferences&, const TraitsDataPtr&, const ContextConstPtr&,
+                             const Manager::RelationshipSuccessCallback&,
+                             const Manager::BatchElementErrorCallback&, const trait::TraitSet&>(
+               &Manager::getWithRelationship),
+           py::arg("entityReferences"), py::arg("relationshipTraitsData").none(false),
+           py::arg("context").none(false), py::arg("successCallback"), py::arg("errorCallback"),
```

Comment:
[question] Is this still needed now were not overloading methods? Is it needed below too?